sdk/job.py

```diff
@@ -28,3 +28,8 @@ class Job:
     errors: List[str]
     result: str = None
     variables: Dict = None
+
+    def __init__(self, runs, **kwargs):
```

Comment:
why is runs the only positionnal / mandatory argument ?

Reply:
Because it seemed to me we allowed jobs to be created without `variables`. To put it differently, it is the only non-nullable `Job` field that is used by the user to create the Job (before submitting it), the other required fields are filled by the API calls when actually submitting the job. Would you agree with that ?

Reply:
Seen offline: use dictionnary to create jobs instead of partial Job object

Reply:
Done